install_config/topics/configuring_for_gce_registry-proc.adoc

```diff
@@ -0,0 +1,285 @@
+////
+Module included in the following assemblies:
+
+install_config/configuring_gce.adoc
+////
+
+= Configuring the {product-title} registry for GCP
```

Comment:
Better to keep the same when calling "GCE" instead of "GCP"

Reply:
+1 on this.